aboutDesigns.css

```diff
@@ -0,0 +1,57 @@
+.topbar{
```

Comment:
I'm pretty sure you moved the repeated css tags (the ones that are the same across multiple files) into styles.css, so you should remove them from the individual css files. If you haven't removed the repeated css style attributes to a single css file that all of the html files import (ex. .topbar{ display: flex;} seems to be repeated across all the css sheets)

Reply:
Do you think you can do the styles.css thing again? (put it in _assets and have it contain all the css tags that are common across all pages, like the top bar, bottom bar, etc) then, you could remove those common tags from the index.css, about.css, etc files, so they wouldn't be repetitive